store/src/sqlite/model.rs

```diff
@@ -687,6 +751,41 @@ impl iroh_bitswap::Store for ModelStore<Sha256a> {
     }
 }
 
+#[async_trait::async_trait]
+impl AccessModelStore for ModelStore<Sha256a> {
```

Comment:
Similar comment here, Could we be explicit about the reason for implementing api crate traits here.